pyodide-build/pyodide_build/pywasmcross.py

```diff
@@ -1,9 +1,5 @@
 #!/usr/bin/env python3
-"""Helper for cross-compiling distutils-based Python extensions.
-
-distutils has never had a proper cross-compilation story. This is a hack, which
-miraculously works, to get around that.
-
+"""
```

Comment:
These are still accurate remarks if we just replace "distutils" with "Python":
```suggestion
"""Helper for cross-compiling Python binary extensions.

Python has never had a proper cross-compilation story. This is a hack, which
miraculously works, to get around that.

```